<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tap_1">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="7" column="1">
            <widget class="QSpinBox" name="OrigenY_spinBox"/>
           </item>
           <item row="11" column="1">
            <widget class="QSpinBox" name="Red_spinBox"/>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="Velocidad_spinBox"/>
           </item>
           <item row="12" column="1">
            <widget class="QSpinBox" name="Green_spinBox"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Destino y:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="DestinoX_spinBox"/>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Rojo:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="Id_spinBox">
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Origen Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Destino x: </string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0" colspan="2">
            <widget class="QPushButton" name="Agre_Final_buttom">
             <property name="text">
              <string>Agregar FInal</string>
             </property>
            </widget>
           </item>
           <item row="19" column="0" colspan="2">
            <widget class="QPushButton" name="Mostrar_buttom">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="18" column="0" colspan="2">
            <widget class="QPushButton" name="Agre_Inicio_Buttom">
             <property name="text">
              <string>Agregar inicio</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QSpinBox" name="Blue_spinBox"/>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="OrigenX_spinBox"/>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Verde:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Azul:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="DestinoY_spinBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Origen X</string>
             </property>
            </widget>
           </item>
           <item row="20" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Ordenar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tap_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLineEdit" name="txt_line">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Id de particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Buscar_Butonn">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="Mostrar_Dates">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Btn_Order">
          <property name="text">
           <string>Ordenar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
